test/controllers/api/distributions_controller_test.rb

```diff
@@ -13,15 +13,20 @@
     assert_response :success
   end
 
-  before(:each) do
-    class << @controller; attr_accessor :prx_auth_token; end
-    @controller.prx_auth_token = token
-  end
-
   describe '#create' do
+
+    before(:each) do
+      class << @controller; attr_accessor :prx_auth_token; end
+      @controller.prx_auth_token = token
+    end
+
     it 'can create a podcast distribution for a series' do
+      template_uri = "/api/v1/audio_version_templates/#{audio_version_template.id}"
+      podcast_uri = 'http://feeder.prx.org/api/v1/podcast/12345'
```

Comment:
won't change anything, but do you want to use this `podcast_uri` variable on line 33 to avoid repeating the full uri there? (that line is untouched in this PR --         `res.update_attributes!(url: 'http://feeder.prx.org/api/v1/podcast/12345'`)